### PR TITLE
Proper "export" header for library consumers

### PR DESCRIPTION
**Pull request by DarkWanderer**

### CODE REVIEW

**Comment by DarkWanderer** [comment]
@X39 I think I got all of the typos this time

File: src/export/sqfvm.cpp [hunk 1]
```diff
@@ -147,23 +148,23 @@ namespace dllexports
     }
 }
 extern "C" {
-    DLLEXPORT_PREFIX void* sqfvm_create_instance(void* user_data, sqfvm_log_callback callback, float max_runtime_seconds)
+    SQLVM_EXPORT void* sqfvm_create_instance(void* user_data, sqfvm_log_callback callback, float max_runtime_seconds)
```
Comment (X39) [inline src/export/sqfvm.cpp:151]: `SQLVM_EXPORT` VS `SQFVM_EXPORT`

File: src/export/sqfvm.cpp [hunk 2]
```diff
@@ -147,23 +148,23 @@ namespace dllexports
...
     {
         return dllexports::create_instance(user_data, callback, max_runtime_seconds, dllexports::ops_set::full);
     }
-    DLLEXPORT_PREFIX void* sqfvm_create_instance_basic(void* user_data, sqfvm_log_callback callback, float max_runtime_seconds)
+    SQLVM_EXPORT void* sqfvm_create_instance_basic(void* user_data, sqfvm_log_callback callback, float max_runtime_seconds)
```
Comment (X39) [inline src/export/sqfvm.cpp:155]: `SQLVM_EXPORT` VS `SQFVM_EXPORT`

File: src/export/sqfvm.cpp [hunk 3]
```diff
@@ -147,23 +148,23 @@ namespace dllexports
...
     {
         return dllexports::create_instance(user_data, callback, max_runtime_seconds, dllexports::ops_set::basic);
     }
-    DLLEXPORT_PREFIX void* sqfvm_create_instance_empty(void* user_data, sqfvm_log_callback callback, float max_runtime_seconds)
+    SQLVM_EXPORT void* sqfvm_create_instance_empty(void* user_data, sqfvm_log_callback callback, float max_runtime_seconds)
```
Comment (X39) [inline src/export/sqfvm.cpp:159]: `SQLVM_EXPORT` VS `SQFVM_EXPORT`

File: src/export/sqfvm.cpp [hunk 4]
```diff
@@ -147,23 +148,23 @@ namespace dllexports
...
     {
         return dllexports::create_instance(user_data, callback, max_runtime_seconds, dllexports::ops_set::none);
     }
-    DLLEXPORT_PREFIX void sqfvm_destroy_instance(void* instance)
+    SQLVM_EXPORT void sqfvm_destroy_instance(void* instance)
     {
         return dllexports::destroy_instance(instance);
     }
-    DLLEXPORT_PREFIX int32_t sqfvm_load_config(void* instance, const char* contents, uint32_t length)
+    SQLVM_EXPORT int32_t sqfvm_load_config(void* instance, const char* contents, uint32_t length)
```
Comment (X39) [inline src/export/sqfvm.cpp:167]: `SQLVM_EXPORT` VS `SQFVM_EXPORT`

File: src/export/sqfvm.cpp [hunk 5]
```diff
@@ -147,23 +148,23 @@ namespace dllexports
...
     {
         return dllexports::create_instance(user_data, callback, max_runtime_seconds, dllexports::ops_set::basic);
     }
-    DLLEXPORT_PREFIX void* sqfvm_create_instance_empty(void* user_data, sqfvm_log_callback callback, float max_runtime_seconds)
+    SQLVM_EXPORT void* sqfvm_create_instance_empty(void* user_data, sqfvm_log_callback callback, float max_runtime_seconds)
     {
         return dllexports::create_instance(user_data, callback, max_runtime_seconds, dllexports::ops_set::none);
     }
-    DLLEXPORT_PREFIX void sqfvm_destroy_instance(void* instance)
+    SQLVM_EXPORT void sqfvm_destroy_instance(void* instance)
```
Comment (X39) [inline src/export/sqfvm.cpp:163]: `SQLVM_EXPORT` VS `SQFVM_EXPORT`

File: src/export/sqfvm.cpp [hunk 6]
```diff
@@ -190,7 +191,7 @@ extern "C" {
             return instance_invalid;
         }
     }
-    DLLEXPORT_PREFIX int32_t sqfvm_call(void* instance, void* call_data, char type, const char* code, uint32_t length)
+    SQLVM_EXPORT int32_t sqfvm_call(void* instance, void* call_data, char type, const char* code, uint32_t length)
```
Comment (X39) [inline src/export/sqfvm.cpp:194]: `SQLVM_EXPORT` VS `SQFVM_EXPORT`

File: src/export/sqfvm.cpp [hunk 7]
```diff
@@ -341,7 +342,7 @@ extern "C" {
             return instance_invalid;
         }
     }
-    DLLEXPORT_PREFIX int32_t sqfvm_status(void* instance)
+    SQLVM_EXPORT int32_t sqfvm_status(void* instance)
```
Comment (X39) [inline src/export/sqfvm.cpp:345]: `SQLVM_EXPORT` VS `SQFVM_EXPORT`

File: src/export/sqfvm.h [hunk 8]
```diff
@@ -54,7 +56,7 @@ extern "C" {
     //         -1 if the instance was null
     //         -2 if preprocessing failed
     //         -3 if parsing failed
-    DLLEXPORT_PREFIX int32_t sqfvm_load_config(void* instance, const char* contents, uint32_t length);
+    SQLVM_EXPORT int32_t sqfvm_load_config(void* instance, const char* contents, uint32_t length);
```
Comment (X39) [inline src/export/sqfvm.h:59]: `SQLVM_EXPORT` VS `SQFVM_EXPORT`

**Review by X39 — CHANGES_REQUESTED** [review]

**Review by X39 — APPROVED** [review]

**Pull request merged**